clap_builder/src/builder/command.rs

```diff
@@ -3125,7 +3125,7 @@ impl Command {
         }
     }
 
-    /// Multiple-personality program dispatched on the binary name (`argv[0]`)
+    /// Sets this program as a "multicall" executable, dispatched on the binary name (`argv[0]`)
```

Comment:
I feel I'm missing something about this clarification as it basically changes it from being a semantic description to reusing the name of the function to identify what this does.

Reply:
I don't find multiple-personality is any different unless you're familiar with a media trope; I'd rather just have the description of what it does there than try to summarize in a confusing way. 

What about something like "Allows program functionality to be selected by `argv[0]` (the name of the binary or link this program is called by)"

Reply:
For myself, I feel like that says less that today with more words while also going into more detail on a specific point that is best left for the more detailed description.

When I read this (note that I don't think I wrote this as multicall support was contributed by someone else), it is familiar to me, not because of media's portrayal of mental health but because I've worked with products where the term "personality" was used to describe the behavior so saying something has "multiple personalities" implies it supports multiple concrete forms.

[Busybox documents it as](https://www.busybox.net/BusyBox.html)

> BusyBox is a multi-call binary. A multi-call binary is an executable program that performs the same job as more than one utility program. That means there is just a single BusyBox binary, but that single binary acts like a large number of utilities. This allows BusyBox to be smaller since all the built-in utility programs (we call them applets) can share code for many common operations.

Instead of "personality" this is talking about "utility program".  This feels too limited as we've found other uses for multicall than busybox like programs, like REPLs.